dm/dm.go

```diff
@@ -122,10 +122,11 @@ func getRegistryProvider() registry.RegistryProvider {
 func newRegistry(URL string) *common.Registry {
 	tFormat := fmt.Sprintf("%s;%s", common.VersionedRegistry, common.CollectionRegistry)
 	return &common.Registry{
-		Name:   util.TrimURLScheme(URL),
-		Type:   common.GithubRegistryType,
-		URL:    URL,
-		Format: common.RegistryFormat(tFormat),
+		Name:           util.TrimURLScheme(URL),
+		Type:           common.GithubRegistryType,
+		URL:            URL,
+		Format:         common.RegistryFormat(tFormat),
+		CredentialName: "default",
```

Comment:
Discussed offline. But if a credential is not found, we fall back to unauthenticated, hence the behaviour is like it was before this CL.